### DISCUSSION TITLE
[Feature Request] Tooltips

### DISCUSSION THREAD

**Discussion by ScottBeeson** [ChristopherKlay/StadiaEnhanced#47] [Ideas & Requests]
Please add tooltips. I didn't know what most of the settings did without clicking them wildly. I still don't know exactly what some of them do.  ¯\\_(ツ)_/¯

**Comment by ChristopherKlay** [comment]
Definitely part of a bigger issue with the menu, together with the menu itself just overflowing at some point if i add more options to it.

Which is actually why a complete re-design of the settings menu is currently the highest spot on my to-do list, which will change the "pop-up and click" style menu into a actual list of options with a short description (including the default value), toggle/dropdown options and such, similar to the actual Stadia settings page.

I feel like most people don't really change those options all that often in the first place and a more detailed menu that's easily expandable, but also still reachable via a single click like the current one, will just be a really good long term addition. I just don't want to push something like that, that could potentially break/create issues and would require completely re-writing translations, directly before christmas, where the situation could come up that something breaks and i'm unable to directly fix it.

It's definitely coming sooner than later however, most likely included in the next update already. There's just a couple of things i want to get done first, to bundle in a slightly bigger update with it (:

For now however, a short description of the menu items;

### Interface
Title|Options|Description
:-:|:-:|:--
Grid Size|2 to 6|Changes the amounts of game "tiles" per row on the homescreen.
Clock|Disabled, Friendslist, Overlay, Both|Displays a clock on the friendslist (shift+tab) or as a overlay on the current game when the friendslist is open in-game.
Filter|Disabled, Toggle, Quick|A filter to hide games from the homescreen. Toggle means you have to click the eye icon at the top right and then show/hide games, while the "Quick" option allows you to always show the hide button.
Pro Label|On, Off|Hides the "Pro" label on games on the homescreen when enabled.
User Gallery|IOn, Off|Hides the "Captures" section at the bottom of the homescreen, which is still accessible via the menu entry.
Category Preview|On, Off|Hides the category tags when hovering over a game on the homescreen.
Popup Effect|On, Off|Removes the effect that slightly zooms into games when hovering over them on the homescreen.
Streaming Mode|On, Off|Makes some things unreadable/blurred (like the friendslist) when streaming your screen.

### Messages
Title|Options|Description
:-:|:-:|:--
Message Preview|On, Off|Hides the "Last message" display in the friendslist.
Quick Reply|On, Off|Hides the quick reply section that sometimes comes up above the input in chats.
Offline Friends|On, Off|Hides offline friends in the friendslist.
Invisible Friends|On, Off|Hides friends that don't share their online status.

### Stream
Title|Options|Description
:-:|:-:|:--
Codec|Default, H264, VP9|Changes the used codec for the stream.
Resolution|Native, 1440p, 2160p|Changes the maximum resolution the stream will be received in.
Autostart Monitor|On, Off|Allows the stream monitor to be started when a game starts by default.

**Comment by ChristopherKlay** [comment]
I just posted a preview of some changes to include these in the actual menu here; #53 (:
